src/steps/os/unix.rs

```diff
@@ -1,21 +1,42 @@
-use crate::error::Error;
+use crate::error::{Error, ErrorKind::*};
 use crate::executor::{CommandExt, RunType};
 use crate::terminal::print_separator;
 use crate::utils::{require, which};
 use directories::BaseDirs;
+use std::env;
 use std::path::Path;
 use std::process::Command;
 
+fn zplug_exists(base_dirs: &BaseDirs) -> bool {
+    let home_exists = match env::var("ZPLUG_HOME") {
```

Comment:
You can use some combination of option methods to avoid matching

Reply:
You're absolutely right—It's late in my timezone. Just pushed changes.